recipes/pycalverter/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pyCalverter" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91e26c5d06dfe1495e5b946a724ae691e57ea634ee4a9529a7952b837c041111
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - calverter
+
+about:
+  home: https://bitbucket.org/bayazee/pycalverter/overview
+  license: GNU General Public (GPL)
+  license_family: LGPL
+  license_file: 
```

Comment:
Please change it to ```license_file: LICENSE.txt```
could you also raise an issue in this project to package the license?

Reply:
Done! Issue is #8557